index.js

```diff
@@ -32,8 +32,19 @@ app.get('/todos', (request, response) => {
   });
 });
 
+// Update the title via server:port/todos/id/new_title
+app.patch('/todos/:id/:title', (request, response) => {
```

Comment:
Yes sure it should be todo, I add the :title after I test it on many entries and I got the same title again for many IDs, I tried to prompt the user to enter the new title but it need interface and a lot of work 😂😂